### ISSUE TITLE
Fix malformed and redundant parameter content

### ISSUE THREAD

**Issue by david-waltermire-nist** [usnistgov/OSCAL#191]
# User Story:

As an user of the SP 800-53 rev4 catalog, I need parameter labels to provide useful information that may appear in a user interface. Furthermore, some choices are not properly structured.

The following examples are less than ideal:

```
<param id="ac-4.17_prm_1">
  <label>SELECTION (one or more)</label>
  <select how-many="one or more">
    <choice>organization, system, application, individual</choice>
  </select>
</param>
```

```
<param id="ac-7_prm_3">
            <label>SELECTION</label>
```

## Goals:

- Determine a way to avoid redundant label entries. Consider making labels optional for selection parameters. (@wendellpiez @david-waltermire-nist)
- Find and update parameters that are not properly structured. (@PCrayton).

## Dependencies:

#190 needs to consider this problem, and document the semantics and proper use of labels. #190 needs to be completed before this issue is completed.

## Acceptance Criteria

- All parameters are properly structured and do not contain redundant information that can be easily derived from the underlying structure and profile content.
- All XML and JSON catalog and profile examples have been corrected to remove these issues.

**Comment by wendellpiez** [comment]
Need to coordinate with @david-waltermire-nist on this issue.

This Issue depends on #190 (parameter usage in FedRAMP profiles) or the reverse? If Dave were available for decisions regarding SP800-53 (not FedRAMP) content, these two Issues could be handled together.

**Comment by wendellpiez** [comment]
We addressed this in the working session 2018-06-22. Labels (`label`) are now optional in parameters, with a documented recommendation that they be used always when other information (such as a selection) is not available. An improved copy of SP800-53 is in PR #201.

**Comment by iMichaela** [comment]
## June 28 Status Meeting Update ##
Addressed in the PR #201. Will be completed when this PR is addressed.

**Comment by iMichaela** [comment]
## July 18 Status Update (Sprint 12 Acceptance Meeting) ##
PR #201 is awaiting review from @david-waltermire-nist and/or @iMichaela

**Comment by david-waltermire-nist** [comment]
This was addressed by PR #201. Closing this issue.
